docs/modules/mechanics/tracking-compass.mdx

```diff
@@ -0,0 +1,65 @@
+---
+id: tracking-compass
+title: Tracking Compass
+---
+
+The Tracking Compass module allows you to override the vanilla compass behavior by pointing
+it to a player of interest. In addition, you can control how the compass selects the target to track.
+
+#### Compass Element
+
+<div className="table-container">
+  | Element | Description |
+  |---|---|
+  | `<compass> </compass>` | Node containing all compass properties used in this map. |
+
+  | Sub-element ||
+  |---|---|
+  | `<player>` | An individual compass property. |
+  | `<flag>` | An individual compass property. |
```

Comment:
I realized these sub-element description is repetitive, is there a better way for me to describe `<player>` and `<flag>`, @Pablete1234?

Reply:
Points to the nearest player passing a filter
Points to a specific flag